app/mailers/instructor_notification_mailer.rb

```diff
@@ -2,27 +2,28 @@
 
 class InstructorNotificationMailer < ApplicationMailer
   def self.send_email(alert)
-    return unless Features.email?
+    # return unless Features.email?
     email(alert).deliver_now
   end
 
   def email(alert)
     @alert = alert
     set_email_parameters
     params = { to: @instructors.pluck(:email),
-               subject: @alert.subject }
-    params[:reply_to] = @alert.sender_email unless @alert.sender_email.nil?
-    unless @alert.sender_email.nil?
-      params[:bcc] = @alert.sender_email # sender_email gives user email of the sender
+               subject: @alert[:details][:subject] }
+    unless @alert[:details][:bcc_to_salesforce] == false
```

Comment:
I suggest using `dig` here rather than a series of `[]` calls, so that it will handle it gracefully if this method is called on an alert record that lacks the expected bbc_to_salesforce field (such as an older record from before the PR).

Reply:
Alright, i surely missed that i will , update this